c-api/compat-5.3.h

```diff
@@ -88,6 +88,19 @@ extern "C" {
 #  define LUA_OPLE 2
 #endif
 
+/* LuaJIT/Lua 5.1 does not have the updated 
+ * error codes for thread status/function returns (but some patched versions do)
+ * define it only if it's not found
+ */
+#if !defined(LUA_ERRGCMM)
```

Comment:
Could use `#ifndef`?

Reply:
Other places in the file prefer `#if !defined`: I followed the style.